src/acvp.c

```diff
@@ -2032,7 +2059,12 @@ static ACVP_RESULT acvp_process_vsid(ACVP_CTX *ctx, char *vsid_url, int count) {
             /*
              * Wait and try again to retrieve the VectorSet
              */
-            acvp_retry_handler(ctx, retry_period);
+            if (acvp_retry_handler(ctx, (unsigned int *)&retry_period, (unsigned int *)&time_waited_so_far, 1)
```

Comment:
Do these two need to be type cast ? Seems like just using & should work since they are defined as unsigned int.
Same below where acvp_retry_handler is called as well.